### ISSUE TITLE
Color contrast errors in site example

### ISSUE THREAD

**Issue by mgifford** [ProseMirror/website#173]
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (a[href$="ref/#inputrules"])
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: ProseMirror basic example - https://prosemirror.net/examples/basic/

Element path: a[href$="ref/#inputrules"]

Snippet: <a href="/docs/ref/#inputrules">Input rules</a>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.2 (foreground color: #0075ff, background color: #ffffff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 131.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.44.0 (axe-core 4.9.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
